packages/web/src/scss/components/PricingPlan/_PricingPlanBody.scss

```diff
@@ -0,0 +1,73 @@
+// 1. Make space for the footer, no matter it's present or not. Since the empty grid rows collapse, there won't be any
+//    undesired vertical space in case the footer is not there.
+//
+// 2. Use `margin-top` instead of `gap` to avoid the gap being added between empty grid rows.
+//
+// 3. Optionally, using the subgrid layout, the pricing plan features can be aligned across pricing plans if they are
+//    all marked as comparable.
+//
+//    👉 Please note that the pricing plans must be wrapped in the Matrix component for this to work. Otherwise, the
+//    body items will be just stacked vertically.
+
+@use '../../tools/typography';
+@use 'theme';
+
+.PricingPlanBody {
+    display: grid;
+    grid-row: body-start / (-1 - theme.$footer-rows); // 1.
+    align-content: start;
+    padding-inline: theme.$padding-x;
+    padding-block: theme.$body-padding-top theme.$body-padding-bottom;
+    color: theme.$body-color;
+
+    &:not(:last-child) {
+        border-bottom: theme.$border-width solid theme.$border-color;
+    }
+}
+
+.PricingPlanBody__featureList:not(:first-child) {
+    margin-top: theme.$body-feature-spacing; // 2.
+}
+
+.PricingPlanBody__featureItem {
+    display: grid;
+    grid-template-columns: auto 1fr;
+    grid-template-areas:
+        'icon title'
+        '. description';
+    gap: theme.$body-feature-row-gap theme.$body-feature-column-gap;
+    align-content: start;
+
+    &:not(:first-child) {
+        margin-top: theme.$body-feature-spacing; // 2.
+    }
+}
+
+.PricingPlanBody__featureTitle {
+    display: contents;
+}
+
+.PricingPlanBody__featureTitle > svg {
```

Comment:
Icons should be green by default.

<img width="298" alt="obrazek" src="https://github.com/user-attachments/assets/d3a33bf1-4966-4be9-9a51-46facf4be201" />